File: TypeScript/ExampleTests/production/Address.ts
export class Address {
  street: SimpleText;
  city: SimpleText;
  state: SimpleText;
  zIP: SimpleText;

  constructor(street: SimpleText, city: SimpleText, state: SimpleText, zIP: SimpleText) {
    this.street = street;
    this.city = city;
    this.state = state;
    this.zIP = zIP;
  }
}

File: Rust/ExampleTests/production/address.rs
use super::*;

#[derive(Debug, Clone, PartialEq)]
pub struct Address {
    pub street: SimpleText,
    pub city: SimpleText,
    pub state: SimpleText,
    pub zip: SimpleText,
}

impl Address {
    pub fn new(street: SimpleText, city: SimpleText, state: SimpleText, zip: SimpleText) -> Self {
        Self { street,  city,  state,  zip }
    }
}
